app/utils/storage-migration.ts

```
@@ -0,0 +1,319 @@
+          error: error instanceof Error ? error.message : String(error),
+        };
+      }
+    }
+
+    const backupString = JSON.stringify(backup);
+    localStorage.setItem("pre-migration-backup", backupString);
+
+    console.log("[StorageMigration] Backup created successfully");
+    return backupString;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)